freki.go

```diff
@@ -331,14 +339,22 @@ func (p *Processor) mangle(
 	packet gopacket.Packet,
 	ip *layers.IPv4,
 	tcp *layers.TCP,
+	udp *layers.UDP,
 	body *gopacket.Payload) error {
 
 	var err error
 	var buffer gopacket.SerializeBuffer
+	var dstPort gopacket.Endpoint
+	switch {
```

Comment:
now that we are adding more protocols, maybe we should explicitly `panic` by default here? Though we should make sure that the panic is `recovered` to unroll the iptable changes.